201-web-app-asev2-create/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aseName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the App Service Environment"
+            }
+        },
+        "aseLocation": { 
+            "type": "string", 
+            "allowedValues": [ 
+                "Central US", 
+                "East US", 
+                "East US 2", 
+                "North Central US", 
+                "South Central US", 
+                "West US",
+                "North Europe", 
+                "West Europe", 
+                "East Asia", 
+                "Southeast Asia", 
+                "Japan East", 
+                "Japan West", 
+                "Brazil South", 
+                "Australia East", 
+                "Australia Southeast", 
+                "West India", 
+                "Central India", 
+                "South India",
+                "Canada Central",
+                "Canada East",
+                "West Central US",
+                "West US 2",
+                "UK West",
+                "UK South",
+                "Korea Central",
+                "Korea South"
+            ], 
+            "metadata": { 
+                "description": "Location of the App Service Environment" 
+            } 
+         }, 
+        "existingVnetResourceId": {

```

Comment:
Unfortunately no - provisioning of App Service Environments requires subnet specified separately from the vnet resource ID.